src/components/wizard/wizard.component.ts

```diff
@@ -347,6 +349,28 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
         return this.steps.toArray()[index];
     }
 
+    /**
+     * If a step in the wizard becomes invalid, all steps sequentially after
+     * it, should become unvisited and incomplete
+     */
+    setNextStepsUnvisited(currentStep: WizardStepComponent, cb?: (step: WizardStepComponent | MarqueeWizardStepComponent) => void): void {
```

Comment:
We don't want this to be a public method, mark it as `private` if it is only used in this class and `protected` if it is also used in marquee wizard